src/Tools/dotnet-dump/Dumper.cs

```diff
@@ -52,8 +52,14 @@ public int Collect(IConsole console, int processId, string output, bool diag, bo
             }
 
             if (processId == 0) {
-                console.Error.WriteLine("ProcessId is required.");
-                return 1;
+                Console.Error.WriteLine("ProcessId is required.");
+                return 0;
```

Comment:
Should return `1` (as before) to don't indicate success?
Same below.

Reply:
Yes, a non-zero return means failure. This should return 1 and the new error below should return 1.